solidity/contracts/bridge/library/Fraud.sol

```diff
@@ -0,0 +1,448 @@
+// SPDX-License-Identifier: MIT
+
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+// ██████████████     ▐████▌     ██████████████
+// ██████████████     ▐████▌     ██████████████
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+//               ▐████▌    ▐████▌
+
+pragma solidity ^0.8.9;
+
+import {BytesLib} from "@keep-network/bitcoin-spv-sol/contracts/BytesLib.sol";
+import {BTCUtils} from "@keep-network/bitcoin-spv-sol/contracts/BTCUtils.sol";
+import {CheckBitcoinSigs} from "@keep-network/bitcoin-spv-sol/contracts/CheckBitcoinSigs.sol";
+import "../../GovernanceUtils.sol";
+import "./EcdsaLib.sol";
+import "../Bridge.sol";
+
+library Fraud {
```

Comment:
What do you think about naming it `Frauds` and reference it from `Bridge` as `Frauds.Data internal frauds`? I'm currently working on `Wallets` library and looks like the plural form fits better as it produces better naming. It would be good to be consistent everywhere.

Reply:
Done in 1dd89b1b03e42f986817f43eb8c9c7b85b4f77ee